project/Settings.scala

```diff
@@ -217,6 +217,7 @@ object Settings {
 
     val LucumaReact = Def.setting(
       deps(
+        "edu.gemini" %%% "lucuma-react-table",
```

Comment:
Reordered these dependencies to get rid of a compiler AssertionError. It is probably related to the ScalablyTyped generated facades having duplicates. A recent change to lucuma-react changed all of the generated facades to use `reactST` as the prefix. I think this changed the issues the scala 3 compiler is having.